### DISCUSSION TITLE
Add the possiblity to used ignoredErrors

### DISCUSSION THREAD

**Discussion by RareDevil** [exceljs/exceljs#1910] [Ideas & Proposals]
I have tried a few things now to get exceljs to add ignore errors like the "Number Stored As Text".
So far, I have worked out that in the XML format you can add the following  XML
```xml
<ignoredErrors>  
  <ignoredError sqref="A1 B2" numberStoredAsText="1"/>  
</ignoredErrors>  
```
And the sheet should ignore the "Number Stored As Text" error message, but so far, I have not been able to work out how to make it work. 

I write here in hopes that someone out there may know how to do this because right now, I'm stuck on this and really wanted to make it work.

More information on what I'm trying to get working can be found here
https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.ignorederror?view=openxml-2.8.1

Also, I have seen this with the DevExtreme datagrid export feature and after reading the code is where I found out what they did to make it work, but it does not directly translate to exceljs sadly.

--- 

I have currently solved my problem here with hard-coding a line into `worksheet-xform.js` or in my case it is just exceljs.js. 
So in the `render` function for `WorkSheetXform` just before `xmlStream.closeNode();` I have added the following line of code, `xmlStream._xml.push("<ignoredErrors><ignoredError sqref=\"A1:XFD1048576\" numberStoredAsText=\"1\" /></ignoredErrors>");` this no way pretty but right now, it works for me. 

It would be nice if there was an ignore error property instead of my code here. 
So, I know this may not be the right format for an idea / proposal ticket but instead of creating a new discussion about this for an idea / proposal I will convert this to it. 

I would say that how this should work could be as following:
Add the property `ignoredError` to the `workbook.properties`.
How `ignoredError` can be structured could be something like this:
```javascript
// To set it for all rows and columns
workbook.properties.ignoredError = {
   numberStoredAsText: true
}
// To set it for just some columns and/or rows
workbook.properties.ignoredError = {
   numberStoredAsText: {
       col: 2,
       row: 2, // if this is set then it will only look for the cel "B2" 
   }
}

// To set it for range of columns and/or rows
workbook.properties.ignoredError = {
   numberStoredAsText: {
      from: {
          row: 3,
          column: 1
      },
      to: {
          row: 5,
          column: 12
      }
   }
}
```
So, each error could only affect a specific cell or the entire spreadsheet.

**Comment by Munaro** [comment]
that would be perfect
